### PR TITLE
[docs] Inomurko/boba on goerli

### PR DESCRIPTION
**Pull request by InoMurko**
## Overview

As Boba on Goerli is being finalized, we can put up docs related to Goerli.

## Changes

Seek Rinkeby, replace by Goerli

## Testing

How do we test docs?

### CODE REVIEW

**Review by souradeep-das — APPROVED** [review]

File: boba_documentation/developer/bridge-l1-and-l2/standard-bridge.md [hunk 1]
```diff
@@ -163,7 +163,7 @@ The Standard bridge allows a one-to-many mapping between L1 and L2 tokens, meani
 | Network | URL                                                          |
 | ------- | ------------------------------------------------------------ |
 | Mainnet | [Mainnet Boba Token List](https://github.com/bobanetwork/boba/blob/develop/packages/boba/register/addresses/addressesMainnet_0x8376ac6C3f73a25Dd994E0b0669ca7ee0C02F089.json) |
-| Rinkeby | [Rinkeby Boba Token List](https://github.com/bobanetwork/boba/blob/develop/packages/boba/register/addresses/addressesRinkeby_0x93A96D6A5beb1F661cf052722A1424CDDA3e9418.json) |
+| Goerli | [Goerli Boba Token List](https://github.com/bobanetwork/boba/blob/develop/packages/boba/register/addresses/addressesGoerli_0x93A96D6A5beb1F661cf052722A1424CDDA3e9418.json) |
```
Comment (boyuan-chen) [inline boba_documentation/developer/bridge-l1-and-l2/standard-bridge.md]: The file name is wrong

File: boba_documentation/developer/developer-start.md [hunk 2]
```diff
@@ -19,9 +19,9 @@ Welcome to Boba. Boba is a compute-focused L2. We believe that L2s can play a un
...
+For most contracts, the deploy experience is exactly like deploying on Ethereum. You will need to have some ETH (or Goerli ETH) on Boba and you will have to change your RPC endpoint to either `https://mainnet.boba.network` or `https://goerli.boba.network`. That's it!
 
-The [Mainnet blockexplorer](https://blockexplorer.boba.network) and the [Rinkeby blockexplorer](https://blockexplorer.rinkeby.boba.network) are similar to Etherscan. The [Mainnet gateway](https://gateway.boba.network) and the [Rinkeby gateway](https://gateway.rinkeby.boba.network) allow you to see your balances and bridge funds, among many other functions.
+The [Mainnet blockexplorer](https://blockexplorer.boba.network) and the [Goerli blockexplorer](https://blockexplorer.goerli.boba.network) are similar to Etherscan. The [Mainnet gateway](https://gateway.boba.network) and the [Goerli gateway](https://gateway.goerli.boba.network) allow you to see your balances and bridge funds, among many other functions.
```
Comment (boyuan-chen) [inline boba_documentation/developer/developer-start.md]: Maybe use `https://testnet.bobascan.com`?

File: boba_documentation/developer/exchange-integration.md [hunk 3]
```diff
@@ -175,8 +175,8 @@ The Standard bridge allows a one-to-many mapping between L1 and L2 tokens, meani
 
 | Network | URL                                                                                                                                                                                  |
 | ------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| Mainnet | [Mainnet Boba Token List](https://github.com/bobanetwork/boba/blob/develop/packages/boba/register/addresses/addressesMainnet\_0x8376ac6C3f73a25Dd994E0b0669ca7ee0C02F089.json) |
-| Rinkeby | [Rinkeby Boba Token List](https://github.com/bobanetwork/boba/blob/develop/packages/boba/register/addresses/addressesRinkeby\_0x93A96D6A5beb1F661cf052722A1424CDDA3e9418.json) |
+| Mainnet | [Mainnet Boba Token List](https://github.com/bobanetwork/boba/blob/develop/packages/boba/register/addresses/addressesMainnet_0x8376ac6C3f73a25Dd994E0b0669ca7ee0C02F089.json) |
+| Goerli | [Goerli Boba Token List](https://github.com/bobanetwork/boba/blob/develop/packages/boba/register/addresses/addressesGoerli_0x93A96D6A5beb1F661cf052722A1424CDDA3e9418.json) |
```
Comment (boyuan-chen) [inline boba_documentation/developer/exchange-integration.md]: The file name is wrong

File: boba_documentation/developer/exchange-integration.md [hunk 4]
```diff
@@ -290,5 +290,5 @@ The Fast bridge allows a one-to-one mapping between L1 and L2 tokens.
...
```
Comment (boyuan-chen) [inline boba_documentation/developer/exchange-integration.md]: The file name is wrong

File: boba_documentation/developer/network-parameters.md [hunk 5]
```diff
@@ -2,23 +2,23 @@
...
 | --------------- | ---------------------------------------------------------------------------------------- |
-| Rinkeby ChainID | 28                                                                                       |
-| Rinkeby RPC     | [https://rinkeby.boba.network](https://rinkeby.boba.network)                             |
-| Gateway         | [https://gateway.rinkeby.boba.network](https://gateway.rinkeby.boba.network)             |
-| Blockexplorer   | [https://blockexplorer.rinkeby.boba.network](https://blockexplorer.rinkeby.boba.network) |
-| Websocket       | [wss://wss.rinkeby.boba.network](wss://wss.rinkeby.boba.network)                   |
+| Goerli ChainID  | 2888                                                                                       |
+| Goerli RPC      | [https://goerli.boba.network](https://goerli.boba.network)                             |
+| Gateway         | [https://gateway.goerli.boba.network](https://gateway.goerli.boba.network)             |
+| Blockexplorer   | [https://blockexplorer.goerli.boba.network](https://blockexplorer.goerli.boba.network) |
```
Comment (boyuan-chen) [inline boba_documentation/developer/network-parameters.md]: Use `https://testnet.bobascan.com`?

**Review by boyuan-chen — CHANGES_REQUESTED** [review]

File: boba_documentation/developer/bridge-l1-and-l2/standard-bridge.md (hunk 1 above)
Comment (InoMurko) [reply boba_documentation/developer/bridge-l1-and-l2/standard-bridge.md]: Uhhhhi see it!

**Pull request merged**